### PR TITLE
DOCSP-26913: Added reference to upsertDocument option

### PR DESCRIPTION
**Pull request by DBirtolo-mdb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-spark-connector/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-26913>
Staging - <https://docs-mongodbcom-staging.corp.mongodb.com/spark-connector/docsworker-xlarge/DOCSP-26913/configuration/write/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?

### CODE REVIEW

File: source/configuration/write.txt [hunk 1]
```diff
@@ -72,11 +72,11 @@ The following options for writing to MongoDB are available:
 
        - ``insert``: insert the data.
        - ``replace``: replace an existing document that matches the
-         ``idFieldList`` value with the new data, or insert the data if
-         no match exists.
+         ``idFieldList`` value with the new data. If no match exists, the
+         behavior depends on ``upsertDocument``.
```
Comment (ccho-mongodb) [inline source/configuration/write.txt:76]: This behavior seems tough to describe more clearly, but maybe by specifying how it depends on "upsertDocument" would make it more clear.

```suggestion
         ``idFieldList`` value with the new data. If no match exists, the
         connector uses the value of  the ``upsertDocument``setting
         to determine whether to perform an upsert.
```

Similar comment for lines 78-79.

File: source/configuration/write.txt [hunk 2]
```diff
@@ -130,6 +130,15 @@ The following options for writing to MongoDB are available:
          guide on the
          :manual:`WriteConcern wtimeout option </reference/write-concern/#wtimeout>`.
 
+   * - ``upsertDocument``
+     - | When true, ``replace`` and ``update`` operations will insert the data
```
Comment (ccho-mongodb) [inline source/configuration/write.txt:134]: Suggestion:
1) I'm not sure if there's a standard for formatting values and operations, but it seems confusing when "true" is unformatted whereas in other instances it is formatted with double backticks. 

2) I think the "replace" and "update" operations should be unformatted here based on how, when they are used to describe operations and not setting values, they are unformatted in other parts of this table.

**Review by ccho-mongodb — APPROVED** [review]
LGTM. Left a couple suggestions that might make it more clear and consistent.

**Review by rozza — APPROVED** [review]
LGTM

**Pull request merged**